Model/Config/Tesco.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Omise\Payment\Model\Config;
+
+use Omise\Payment\Model\Config\Config;
+
+class Tesco extends Config
+{
+    /**
+     * @var string
+     */
+    const CODE = 'omise_offsite_tesco';
```

Comment:
@jacekstan IIRC we reached to the conclusion that `offsite` doesn't sound fit with Tesco Bill Payment (ambiguous) its flow has no redirection out from the store. 

Should it be `omise_apm_tesco`, `omise_offline_tesco`, or something?